config/manager/kustomization.yaml

```diff
@@ -1,16 +1,2 @@
 resources:
 - manager.yaml
-
-generatorOptions:
-  disableNameSuffixHash: true
-
-configMapGenerator:
-- files:
-  - controller_manager_config.yaml
-  name: manager-config
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-images:
-- name: controller
-  newName: quay.io/openstack-k8s-operators/octavia-operator
-  newTag: latest
```

Comment:
I think this gets re-added when calling `make bundle`. maybe good to run it and add it